list-files.sh

```diff
@@ -5,9 +5,31 @@ script_dirpath="$(cd "$(dirname "${0}")" && pwd)"
 
 fd_base_cmd="fd --follow --hidden --color=always"
 
+# Common project directories to exclude
+common_excludes="\
+    -E 'node_modules' \
+    -E '.git' \
+    -E 'dist' \
+    -E 'build' \
+    -E 'target' \
+    -E '.next' \
+    -E '.nuxt' \
+    -E 'coverage' \
+    -E '.pytest_cache' \
+    -E '__pycache__' \
+    -E '.venv' \
+    -E 'vendor' \
+    -E '.tox' \
+    -E '.mypy_cache' \
+    -E '.ruff_cache' \
+    -E '.turbo' \
+    -E 'out' \
+    -E '.parcel-cache' \
+    -E '.terraform'"
+
 # If the user passes in a '-o' argument, we only list the contents of the current directory
 if [ "${1:-}" = "-o" ]; then
-    ${fd_base_cmd} --max-depth 1 .
+    eval "${fd_base_cmd} --max-depth 1 ${common_excludes} ."
```

Comment:
Do we need the `eval`s here and below? Generally I prefer to avoid `eval` when possible

Reply:
Ah also, we can't just add `common_excludes` here and exit anymore, since this won't add back the project-specific directories.